hugr-core/src/extension/op_def.rs

```diff
@@ -611,7 +611,7 @@ pub(super) mod test {
         assert_eq!(def.misc.len(), 1);
 
         let reg =
-            ExtensionRegistry::try_new([PRELUDE.to_owned(), EXTENSION.to_owned(), e]).unwrap();
+            ExtensionRegistry::try_new([PRELUDE.clone(), EXTENSION.clone(), e.into()]).unwrap();
```

Comment:
Note that for all of these `to_owned` was still valid.
I just think that `clone`-ing the `Arc` is more explicit now, that we can use that.